src/graphics/engine/engine.h

```diff
@@ -1406,6 +1406,13 @@ class CEngine : public CSingleton<CEngine>
 
     Texture         m_shadowMap;
 
+    struct
```

Comment:
The linter complains about the anonymous struct too, although I personally think it's cleaner that way... Maybe just make it `struct PendingDebugDraw {...} m_pendingDebugDraws;`